app/helpers/comments_helper.rb

```diff
@@ -32,6 +32,24 @@ def link_to_comment_ultimate_parent(comment)
     end
   end
 
+  def comment_link_with_commentable_name(comment)
+    ultimate_parent = comment.ultimate_parent
+    commentable_name = ultimate_parent&.commentable_name
+    text = case ultimate_parent.class.to_s
+           when "Work"
+             t("comments_helper.comment_link_with_commentable_name.on_work_html", title: commentable_name)
+           when "AdminPost"
+             t("comments_helper.comment_link_with_commentable_name.on_admin_post_html", title: commentable_name)
+           else
+             if ultimate_parent.is_a?(Tag)
```

Comment:
Could we combine these lines to `elsif ultimate_parent.is_a?(Tag)`?

Reply:
It won't work if this is a `case` statement. I can make this a plain if/elsif/else if we don't mind it being different from the `link_to_comment_ultimate_parent` method above.

Reply:
I didn't know that, I think this way works just fine.